tests/test-http-stream-finish.lua

```diff
@@ -1,7 +1,16 @@
 local http = require("http")
 
 require('tap')(function(test)
-  test('http stream end', function(expect)
+
```

Comment:
Yes, I've commented because is a test for a bug. I will remove the comment with the fix of the http bug. What do you think?